library/src/scripts/messages/messageStyles.ts

```diff
@@ -95,6 +95,25 @@ export const messagesClasses = useThemeCache(() => {
         zIndex: 20,
     });
 
+    const innerWrapper = style("innerWrapper", {
+        $nest: {
+            "&&": {
+                flexDirection: "row",
+                maxWidth: 920,
```

Comment:
Where does this number come from? Is it something that should be a calculated value (maybe based off of our column widths)?

In any case it needs to be a variable.

We want to ensure that if we change something like the main column width, that stuff like this is affected too.

Reply:
removed max-width: static number from container css